openff/nagl/tests/test_lookups.py

```diff
@@ -99,6 +99,19 @@ def test_lookup_with_different_connectivity(self, lookup_table):
         assert_allclose(properties.numpy(), np.array([-0.103, 0.234, -0.209, -0.209, 0.096, 0.096, 0.096]))
 
     def test_lookup_long(self, lookup_table):
+        from openff.toolkit import __version__ as toolkit_version
+        from openff.utilities import has_package
+
+
         mol = Molecule.from_smiles(341 * "C")
-        with pytest.raises(KeyError, match="failed to generate an InChI"):
-            lookup_table.lookup(mol)
+
+        # toolkit >= 0.16.9 with RDKit supports "large" InChI
+        if toolkit_version >= "0.16.9" and not has_package("openeye"):
```

Comment:
Is this comparison safe without casting these as version objects?

Reply:
Believe it or not, it's pretty safe. I'm not sure when this was introduced, but strings that look like version numbers have some magic methods implemented


```python
>>> '1.0.0' > '0.17.0' > '0.16.9'
True
```

if there's a reason to use `packaging.version.Version` we can go for it since I think that's already packaged somewhere in our ecosystem